packages/quantic/force-app/main/default/lwc/quanticSort/e2e/quanticSort.e2e.ts

```diff
@@ -0,0 +1,75 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+
+const newestDateSort = 'date descending';
+const defaultSortLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test;
+  if (useCase.value === useCaseEnum.search) {
+    test = fixtures[useCase.value] as typeof testSearch;
+  } else {
+    test = fixtures[useCase.value] as typeof testInsight;
+  }
```

Comment:
Why do you need the `as typeof` ?
```suggestion
  const test = fixtures[useCase.value];
```

Reply:
I won't fix it now, as it's bit complicated. Mehdi will review and fix them later

Reply:
@mmitiche any insight here?